test/java/seedu/addressbook/util/TypicalPersons.java

```diff
@@ -50,4 +57,19 @@ public AddressBook getTypicalAddressBook() {
         return ab;
     }
 
+    public List<ReadOnlyPerson> getListWithAllPersons() {
```

Comment:
Also, not sure if you need this method. Can you use `getTypicalPersons()` instead, and convert it to a list using Arrays.asList()?